acceptancetests/assess_upgrade_series.py

```diff
@@ -81,6 +81,10 @@ def reboot_machine(client, machine):
             raise e
         log.info("Ignoring `juju ssh` exit status after triggering reboot")
 
+    # Wait a bit so that we do not detect the started condition
+    # *before* Juju actually reports it as "down".
+    time.sleep(5)
```

Comment:
Because of commodity hardware, I have no faith in 5 seconds. Is there a way to either wait for its state to change to "down" or just pad out 5 seconds to longer?